internal/command/deploy/machines_deploymachinesapp.go

```diff
@@ -485,261 +475,6 @@ func (md *machineDeployment) updateUsingBlueGreenStrategy(ctx context.Context, u
-				statuslogger.StatusRunning,
-				"Updating %s",
-				md.colorize.Bold(fmtID),
-			)
-		}
-		statusFailure := func(err error) {
-			if errors.Is(err, context.Canceled) {
-				statuslogger.LogfStatus(eCtx,
-					statuslogger.StatusFailure,
-					"Machine %s update %s",
-					md.colorize.Bold(fmtID),
-					md.colorize.Red("canceled while it was in progress"),
-				)
-			} else {
-				statuslogger.LogfStatus(eCtx,
-					statuslogger.StatusFailure,
-					"Machine %s update %s: %s",
-					md.colorize.Bold(fmtID),
-					md.colorize.Red("failed"),
-					err.Error(),
-				)
-			}
-		}
-		statusSuccess := func() {
-			statuslogger.LogfStatus(eCtx,
-				statuslogger.StatusSuccess,
-				"Machine %s update %s",
-				md.colorize.Bold(fmtID),
-				md.colorize.Green("succeeded"),
-			)
-		}
-
-		updatesPool.Go(func(_ context.Context) error {
-			statusRunning()
-			if err := md.updateMachine(eCtx, e); err != nil {
-				tracing.RecordError(span, err, "failed to update machine")
-				statusFailure(err)
-				return err
-			}
-			statusSuccess()
-			return nil
-		})
-	}
-
-	return updatesPool.Wait()
-}
-
-func (md *machineDeployment) updateUsingRollingStrategy(parentCtx context.Context, updateEntries []*machineUpdateEntry) error {
-	parentCtx, span := tracing.GetTracer().Start(parentCtx, "rolling", trace.WithAttributes(attribute.String("strategy", md.strategy)))
-	defer span.End()
-
-	sl := statuslogger.Create(parentCtx, len(updateEntries), true)
-	defer sl.Destroy(false)
-
-	// Rolling strategy
-	slices.SortFunc(updateEntries, func(a, b *machineUpdateEntry) int {
-		return cmp.Compare(a.leasableMachine.Machine().ID, b.leasableMachine.Machine().ID)
-	})
-
-	// Group updates by process group
-	entriesByGroup := lo.GroupBy(updateEntries, func(e *machineUpdateEntry) string {
-		return e.launchInput.Config.ProcessGroup()
-	})
-
-	startIdx := 0
-	groupsPool := pool.New().
-		WithErrors().
-		WithMaxGoroutines(rollingStrategyMaxConcurrentGroups).
-		WithContext(parentCtx).
-		WithCancelOnError()
-
-	for group, entries := range entriesByGroup {
-		entries := entries
-		startIdx += len(entries)
-		groupsPool.Go(func(ctx context.Context) error {
-			return md.updateEntriesGroup(ctx, group, entries, sl, startIdx-len(entries))
-		})
-	}
-
-	return groupsPool.Wait()
-}
-
-func (md *machineDeployment) updateEntriesGroup(parentCtx context.Context, group string, entries []*machineUpdateEntry, sl statuslogger.StatusLogger, startIdx int) error {
-	parentCtx, span := tracing.GetTracer().Start(parentCtx, "update_entries_in_group", trace.WithAttributes(
-		attribute.Int("start_id", startIdx),
-		attribute.String("group", group),
-		attribute.Int("max_unavailable", int(md.maxUnavailable)),
-	))
-	defer span.End()
-
-	var poolSize int
-	switch mu := md.maxUnavailable; {
-	case mu >= 1:
-		poolSize = int(mu)
-	case mu > 0:
-		poolSize = int(math.Ceil(float64(len(entries)) * mu))
-	default:
-		return fmt.Errorf("Invalid --max-unavailable value: %v", mu)
-	}
```

Comment:
I can't find this functionality on the new code 👎  seems it is gone

Reply:
added it back :)